cmake/SetupRUClient.cmake

```diff
@@ -0,0 +1,132 @@
+function(setup_ruclient)
```

Comment:
Yes, there are plans to do so. For now, it stays private.

Reply:
Eh, okay, I guess I will rewrite this part under the assumption that someone with the private repo access checked it out at `/extern`.

Reply:
I managed to build `readuntil_fake` on its own locally, but RawHash's relevant part doesn't compile due to mismatch with the state of the current `readuntil_fake`'s main branch:

```console
/home/adamant/RawHash/src/main.cpp:174:59: error: 'class ru_client::DeviceServiceClient' has no member named 'get_calibration'
  174 |                         auto calibrations = device_client.get_calibration(1, n_channels);
```

```console
/home/adamant/RawHash/src/main.cpp:185:25: error: 'ReportNumSamplesBehindCallback' was not declared in this scope
  185 |                         ReportNumSamplesBehindCallback report_num_samples_behind_cb(acquisition_client, 10); // todo1: n_channels instead of 10
      |                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

```console
/home/adamant/RawHash/src/rawhash_ruclient.hpp:70:26: error: 'virtual ru_client::DecisionMaker::Decision RawHashDecisionMaker::decide(const ru_client::ReadIdentifier&, const ru_client::DecisionMaker::ChunkType&, uint32_t)' marked 'override', but does not override
   70 |         virtual Decision decide(ReadIdentifier const& read_ident, ChunkType const& chunk, uint32_t chunk_idx) override;
      |                          ^~~~~~
```

Etc.